browser-switch/build.gradle

```diff
@@ -46,6 +83,9 @@ artifacts {
 }
 
 signing {
+    required {
```

Comment:
This needs to be removed before release, I assume?

Reply:
I think it's fine. This snippet came from braintree_android originally.